mysql-test/suite/rpl/my.cnf

```diff
@@ -1,8 +1,4 @@
 # Use settings from rpl_1slave_base.cnf
 # add setting to connect the slave to the master by default
 !include rpl_1slave_base.cnf
-!include include/default_client.cnf
-
-
-[mysqld.2]
-
+# Override specific server settings using [mariadb-x.y] option group from test.cnf file
```

Comment:
If we're going to add this comment snippet to mention how to override (which I like!), we should probably be more complete about it. That is, to briefly define `x` and `y`, and also add a little example